pkgs/tools/audio/goxlr-utility/default.nix

```diff
@@ -5,34 +5,35 @@
 , pkg-config
 , libpulseaudio
 , dbus
-, speechd-minimal
+, openssl
+, speechd
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "goxlr-utility";
-  version = "1.1.1";
+  version = "2024-08-06";
```

Comment:
```suggestion
  version = "1.1.1-unstable-2024-08-06";
```

Reply:
Ah, that's interesting, I had seen messages about "if it's not a versioned release you should version with the date the commit is from", but had missed this part:

```
If a package is a commit from a repository without a version assigned, then the version attribute should be the latest upstream version preceding that commit, followed by -unstable- and the date of the (fetched) commit. The date must be in "YYYY-MM-DD" format.
```